catalogue/engage.py

```diff
@@ -43,7 +46,9 @@ def git_query(repo_path, commit_changes=False):
     except InvalidGitRepositoryError:
         raise InvalidGitRepositoryError("provided code directory is not a valid git repository")
 
-    if repo.is_dirty():
+    untracked = [f for f in repo.untracked_files if catalogue_dir not in f]
```

Comment:
What if one of the user has an untracked file whose name contains the same text as the `catalogue_dir` string but isn't in that directory?